integration_test.go

```diff
@@ -283,3 +286,89 @@ func TestDefaultValue(t *testing.T) {
 		return result.ID > 0
 	}, "ID should be greater than 0")
 }
+
```

Comment:
Can we add `Go Doc Comments` above every test function that helps understand which feature of gorm is being tested.